hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenCreateValidator.java

```diff
@@ -188,7 +187,5 @@ public void validateAssociation(
                 entitiesConfig.limitTokenAssociations()
                         && account.numberAssociations() + 1 > tokensConfig.maxPerAccount(),
                 TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED);
-        validateTrue(
-                tokenRelStore.get(account.accountId(), token.tokenId()) == null, TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT);
```

Comment:
I don't understand why this line is deleted?

Reply:
The `validateAssociation` is only called at times when it shouldn't be possible to already have an existing `tokenRel` but after thinking it through it is better to have this check, so I reverted the change.